crates/re_types_builder/src/report.rs

```diff
@@ -76,12 +76,12 @@ impl Report {
         let mut errored = false;
 
         while let Ok(warn) = self.warnings.try_recv() {
-            eprintln!("Warning: {warn}");
+            re_log::warn!("{warn}");
         }
 
         while let Ok(err) = self.errors.try_recv() {
             errored = true;
-            eprintln!("Error: {err}");
+            re_log::error!("{err}");
```

Comment:
I don't like this change.

It makes the line longer, but to what benefit? The time stamp is not useful, nor is the crate name.

If you want colored text, you can get it with other means.